src/com/sbezboro/standardplugin/listeners/PlayerChatListener.java

```diff
@@ -14,99 +14,115 @@
+            new AutoHelpTopic("Groups",
+                    new String[]{".*(how|can you|can we|can i).* claim.*",
+                            ".*(how|can you|can we|can i).* own land.*",
+                            ".*(how|can you|can we|can i).* protect.*",
+                            ".*(how|can you|can we|can i).* lock.*",
+                            ".* faction.*"},
+                    new String[]{
+                            ChatColor.DARK_AQUA + "Hey " + ChatColor.BOLD + PLAYER_NAME_REPLACE +
+                                    ChatColor.DARK_AQUA + ", need help with our custom plugin " +
+                                    ChatColor.ITALIC + "Groups" + ChatColor.DARK_AQUA + "?",
+                            ChatColor.DARK_AQUA + "Try clicking on this link: " + ChatColor.UNDERLINE +
+                                    "standardsurvival.com/help"
+                    }),
+
+            new AutoHelpTopic("PVP protection",
+                    new String[]{".*on peaceful.*",
+                            ".*los(e|ing) hunger.*"},
+                    new String[]{
+                            String.valueOf(ChatColor.DARK_AQUA) + ChatColor.BOLD + PLAYER_NAME_REPLACE +
+                                    ChatColor.DARK_AQUA + ", you start off with one hour of" +
+                                    " PVP and hunger protection.",
+                            ChatColor.DARK_AQUA + "Don't worry, you will receive several warnings" +
+                                    " before it expires."
+                    }),
+
+            new AutoHelpTopic("Generic",
+                    new String[]{".*help.*",
+                            ".*how (do|can) (you|we|i).*",
+                            ".*i(['`\"]?m| am) new.*"},
+                    new String[]{
+                            ChatColor.DARK_AQUA + "Hey " + ChatColor.BOLD + PLAYER_NAME_REPLACE +
+                                    ChatColor.DARK_AQUA + ", need help? Try clicking on this link:",
+                            String.valueOf(ChatColor.DARK_AQUA) + ChatColor.UNDERLINE + "standardsurvival.com/help"
+                    })};
+
+    public PlayerChatListener(StandardPlugin plugin) {
+        super(plugin);
+    }
+
+    @EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
+    public void onPlayerChat(AsyncPlayerChatEvent event) {
+        StandardPlayer player = plugin.getStandardPlayer(event.getPlayer());
+
+        String message = event.getMessage();
+
+        if (player.getTimeSpent() < 600) {
+            for (int i = 0; i < helpTopics.length; i++) {
+                if (helpTopics[i].matches(message)) {
+                    helpTopics[i].sendHelpMessage(player);
+                    return;
+                }
+            }
+        }
+        boolean sendWarning = false;
+        char star = ("*").charAt(0);
+        for (String str : plugin.getConfig().getStringList("muted-words")) {
```

Comment:
I use a custom config system in both plugins rather than accessing the Config directly (see the `StandardConfig`) class, all config settings should be defined there, then exposed through `StandardPlugin`